### PR TITLE
Element\DateTime format fix

### PR DESCRIPTION
**Pull request by Kwido**
The static format is now being used rendering the set format useless.

### CODE REVIEW

File: test/Element/MonthTest.php [hunk 1]
```diff
@@ -17,6 +17,7 @@ class MonthTest extends TestCase
     public function testProvidesInputSpecificationThatIncludesValidatorsBasedOnAttributes()
     {
         $element = new MonthElement('foo');
+        $element->setFormat('Y-m');
```
Comment (froschdesign) [inline test/Element/MonthTest.php]: The format setting is wrong, because the `Month` element has already this format. (See also #170)

Comment (froschdesign) [inline test/Element/MonthTest.php]: `protected $format = self::DATETIME_FORMAT;` is missing in `Zend\Form\Element\Month`

Comment (Kwido) [inline test/Element/MonthTest.php]: Yeah I noticed that, I had to set the format due to the missing field. Was in kinda of a rush.

**Comment by froschdesign** [comment]
Related to #168

**Comment by weierophinney** [comment]
Thanks, @Kwido

**Pull request closed without merging**